runner/ansible/vars/azure_env.yml

```diff
@@ -4,3 +4,11 @@ expected:
   "1.1.1": "30000"
   "1.2.1": "true"
   "1.3.5": "60"
+  "1.1.2": "36000"
+  "1.1.3": "20"
+  "1.1.3_runtime": "20"
```

Comment:
Do we need this `1.1.3_runtime` variable? It looks like a leftover

Reply:
Correct, will remove.